draft-ietf-quic-transport.md

```diff
@@ -5753,13 +5753,184 @@ DecodePacketNumber(largest_pn, truncated_pn, pn_nbits):
    return candidate_pn
 ~~~
 
+# Overview of Security Properties {#security-properties}
+
+A complete security analysis of QUIC is outside the scope of this document.  In
+this appendix, we provide an informal description of the desired security
+properties as an aid to implementors and to help guide protocol analysis.
+
+We cover properties of the handshake, general transport, and migration
+separately.
+
+## Handshake {#handshake-properties}
```

Comment:
Maybe a 0-RTT section as well?

Reply:
Makes sense, good thinking! 
Probably others that might be interesting, too, not sure if it's worth adding a bunch of TBD sections, but definitely should have an issue covering writing up the 0-RTT section.